lib/src/qibla.dart

```diff
@@ -4,7 +4,7 @@ import 'internal/qibla_util.dart';
 class Qibla {
   static final Coordinates MAKKAH = Coordinates(21.4225241, 39.8261818);
 
-  double _direction;
+  late double _direction;
```

Comment:
Are we certain that calling QiblaUtil.calculateQiblaDirection(coordinates) won't return null?, as Coordinates class validation is disabled (via validate param) by default;

Reply:
Yes, Coordinates doesn't return null if it is invalid and validation is disabled.